assets/src/components/liveVehicleCard.tsx

```diff
@@ -0,0 +1,28 @@
+import { Socket } from "phoenix"
+import React, { useContext } from "react"
+import { ReactElement } from "react"
+import { SocketContext } from "../contexts/socketContext"
+import useVehicleForId from "../hooks/useVehicleForId"
+import { isVehicle } from "../models/vehicle"
+import { Vehicle } from "../realtime"
+import VehicleCard from "./vehicleCard"
+
+const LiveVehicleCard = ({
```

Comment:
I roughly based this off https://github.com/mbta/skate/blob/master/assets/src/components/propertiesPanel.tsx

Reply:
Some of the complexity with the VPP code, including the way it separately uses the `useVehicle` hook independently of other vehicle state, is actually due to the fact that it needs to keep working if you navigate to a page like settings that doesn't usually cause any vehicle information to be loaded. I think in this case since the card will only ever be appearing in the context of a map that already has the vehicle on it you could just pass the vehicle in via props.

Reply:
Thanks for drawing attention to this weirdness! Passing `selectedVehicle` as a prop wasn't live updating the data, but I added a `useEffect` hook so that when the search results are updated, the `selectedVehicle` is updated too.